tracer/build/_build/Build.Steps.cs

```diff
@@ -602,6 +604,17 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
                 var workingDirectory = ArtifactsDirectory / $"linux-{LinuxArchitectureIdentifier}";
                 EnsureCleanDirectory(workingDirectory);
 
+                if (!IsArm64)
+                {
+                    var tracerNativeFile = MonitoringHomeDirectory / "Datadog.Trace.ClrProfiler.Native.so";
+                    MoveFileToDirectory(tracerNativeFile, MonitoringHomeDirectory / "tracer");
```

Comment:
We _may_ want/need to rename this to avoid confusion (which I think it ok to do) but I can't remember the details. @tonyredondo or @zacharycmontoya may remember the plan around this better than me

Reply:
In the architecture doc that we dug up, it looks like we proposed the name `Datadog.AutoInstrumentation.Tracer.Native.dll` . What do you think? That sounds ok to me

Reply:
As discussed offline, I'll go for `Datadog.Tracer.Native.so`